docs/AppSettingsSqlDss.md

```diff
@@ -0,0 +1,84 @@
+# Sql Server Data source and AIO DSS reference data sink - App Settings Guide
+
+## Configuration Guide
+
+The appsettings.json example below is configured to poll the following SQl based endpoints based on a SQL query definition and send the resulting query to AIO's Distributed Data Store at the configured `key`. Every update to the key value is complete, meaning we replace its entire contents on every run.
```

Comment:
nit: SQl -> SQL

Reply:
Also added a note on the format we being JSON Lines, just below this section in the doc.